apis/mlops/scheduler/scheduler.proto

```diff
@@ -425,6 +425,7 @@ message PipelineVersionState {
     PipelineTerminate = 5;
     PipelineTerminating = 6;
     PipelineTerminated = 7;
+    PipelineRebalancing = 8;
```

Comment:
With the introduction of this new state, I believe some of the state transitions in `pkg/store/pipeline/store.go` may also need to be updated. At a minimum, the `PipelineRebalancing` state needs to be considered as a "running" state [here](https://github.com/lc525/seldon-core/blob/6db0eafc991db056f573d838c77047cfa41378c8/scheduler/pkg/store/pipeline/store.go#L305) for the purposes of PipelineGw rebalancing logic. Would you be able to take a look?